onnxruntime/core/providers/webnn/builders/impl/split_op_builder.cc

```diff
@@ -179,6 +142,23 @@ bool SplitOpBuilder::IsOpSupportedImpl(const InitializedTensorSet& initializers,
       }
     }
   }
+
+  if (split.size()) {
```

Comment:
```suggestion
  if (!split.empty()) {
```

I know it's technically legal to say `if (split.size())`, because the `size_t` is coerced to a `bool`ean, but it's rather irregular, and it can introduce new build warnings on stricter warning settings.